auth-web/src/routes/router.ts

```diff
@@ -134,7 +135,12 @@ export function getRoutes (): RouteConfig[] {
       path: '/account/:orgId/settings',
       name: 'account-settings',
       component: accountSettings,
-      meta: { requiresAuth: true, requiresProfile: true, requiresActiveAccount: true },
+      meta: {
+        requiresAuth: true,
+        requiresProfile: true,
+        requiresActiveAccount: true,
+        allowOnAccountFreeze: true
```

Comment:
allowOnAccountFreeze - any way we can tie this with permissions? making it backend driven?

Reply:
we can make a mapper in the frontend with the route names and permission values, but it still be a dependency with frontend when we add new permission or route changes.
which will be a small effort, but can get ignored sometimes.
if its ok, I can work like that

Reply:
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/5743